.templates/domoticz/service.yml

```diff
@@ -1,16 +1,15 @@
 domoticz:
   container_name: domoticz
-  image: lscr.io/linuxserver/domoticz:latest
+  image: domoticz/domoticz:stable
+  restart: unless-stopped
+  environment:
+    - TZ=${TZ:-Etc/UTC}
+    # - LOG_PATH=/opt/domoticz/userdata/domoticz.log
+    # - EXTRA_CMD_ARG=
   ports:
     - "8083:8080"
-    - "6144:6144"
-    - "1443:1443"
+    - "1443:443"
   volumes:
-    - ./volumes/domoticz/data:/config
-  restart: unless-stopped
-  environment:
-    - PUID=1000
-    - PGID=1000
-    # - TZ=Etc/UTC
-    # - WEBROOT=domoticz
-
+    - ./volumes/domoticz:/opt/domoticz/userdata
+  x-devices:
```

Comment:
YAML needs `/*` and `*/`

Reply:
I assume you mean the `x-` prefix. This is part of the [compose spec](https://github.com/compose-spec/compose-spec/blob/master/11-extension.md) so it's valid syntax.

Reply:
To clarify (from our discussion on Discord) you were lamenting the absence of block (multi-line) comments from the YAML spec.